projects/admin/src/app/record/detail-view/entities-detail-view/local/local-organisation-detail-view/local-organisation-detail-view.component.html

```diff
@@ -0,0 +1,118 @@
+<!--
+  RERO ILS UI
+  Copyright (C) 2019-2023 RERO
+  Copyright (C) 2019-2023 UCLouvain
+
+  This program is free software: you can redistribute it and/or modify
+  it under the terms of the GNU Affero General Public License as published by
+  the Free Software Foundation, version 3 of the License.
+
+  This program is distributed in the hope that it will be useful,
+  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+  GNU Affero General Public License for more details.
+
+  You should have received a copy of the GNU Affero General Public License
+  along with this program.  If not, see <http://www.gnu.org/licenses/>.
+-->
+<dl class="row mb-0">
+  <!-- NAME -->
+  <ng-container *ngIf="record.name">
+    <dt class="col-sm-3 offset-sm-2 offset-md-0 label-title" translate>
+      Name
+    </dt>
+    <dd class="col-sm-7 col-md-8 mb-0">
```

Comment:
as all `dt` and `dd` use the same classes, it should better to use a scss with `@extends` notation. HTML code will be smoother to read and smaller.